basis/concurrency/distributed/distributed-docs.factor

```diff
@@ -14,22 +14,22 @@ ARTICLE: "concurrency.distributed.example" "Distributed Concurrency Example"
 "The code to run the server is:"
 { $code
   "USING: io.servers ;"
-  "9000 local-server <node-server> start-server drop"
+  "9000 local-server start-node"
 }
 "The code to start the thread is:"
 { $code
     "USING: concurrency.messaging threads ;"
     ": log-message ( -- ) receive . flush log-message ;"
-    "[ log-message ] \"logger\" spawn dup name>> register-remote-thread"
+    "[ log-message ] \"logger\" [ spawn ] keep register-remote-thread"
 }
```

Comment:
Isn't this example wrong? `register-remote-thread` has stack effect `( thread name -- )` but you pass it two thread instances.

Reply:
Umm... See the `keep` call? It keeps the "logger" string, doesn't it? Which is the thread's name... Am I right?

Reply:
You are right. My factor skills are getting rusty.

Reply:
I like
```factor
[ log-message ] "logger" [ spawn ] [ register-remote-thread ] bi
```
for symmetry but that's just style :)

Reply:
I can understand your argument, but most times I prefer shorter text, even if it saves only a couple of characters. Maybe I'll change my preference over time.